OWNERS_ALIASES

```diff
@@ -508,18 +483,14 @@ aliases:
     - liggitt # Auth
     - luxas # Cluster Lifecycle
     - marosset # Windows
-    - mattfarina # Apps
     - msau42 # Storage
-    - mwielgus # Autoscaling
```

Comment:
That leaves the autoscaling area without any approver.

Reply:
reverting this change. thanks @mwielgus